crates/bitcoin-da/src/service.rs

```diff
@@ -424,28 +485,20 @@ impl DaService for BitcoinService {
 
     async fn send_transaction(
         &self,
-        blob: &[u8],
+        _blob: &[u8],
     ) -> Result<<Self as DaService>::TransactionId, Self::Error> {
-        let fee_sat_per_vbyte = self.get_fee_rate().await?;
-        self.send_transaction_with_fee_rate(blob, fee_sat_per_vbyte)
-            .await
+        unimplemented!("Use send_tx_no_wait instead")
```

Comment:
why?


use send_tx_no_wait and wait on the receiver :D

Reply:
Thats right, one can wait on the receiver anywhere in the code, not to .await at the call cite